src/content/docs/integrations/kubernetes-integration/installation/install-kubernetes-integration-using-helm.mdx

```diff
@@ -32,674 +34,258 @@ To install the Kubernetes integration using Helm, you will need your New Relic a
+    available configuration options.
+
   </Collapser>
 
   <Collapser
-    className="freq-link"
-    id="install-helm2"
-    title="Helm 2"
+      className="freq-link"
+      id="install-helm2"
+      title="Helm 2 using newrelic-infrastructure"
   >
     To install using Helm 2:
 
     1. Set the cluster where you want to install the agent:
 
-       ```
-       kubectl config set-cluster <var>DESIRED_CLUSTER</var>
-       ```
-
-       To see the available clusters, run `kubectl config get-clusters`
-    2. If your cluster uses role-based access, create a service account for `tiller` (Helm's service which runs inside the Kubernetes cluster) using a Kubernetes manifest file:
-
-       1. Paste the following in a manifest file `rbac-config.yaml`:
-
-          ```
-          apiVersion: v1 
-          kind: ServiceAccount 
-          metadata: 
-            name: tiller 
-            namespace: kube-system 
-          --- 
-          apiVersion: rbac.authorization.k8s.io/v1 
-          kind: ClusterRoleBinding 
-          metadata: 
-            name: tiller 
-          roleRef: 
-            apiGroup: rbac.authorization.k8s.io 
-            kind: ClusterRole 
-            name: cluster-admin 
-          subjects: 
-            - kind: ServiceAccount 
-              name: tiller 
-              namespace: kube-system
-          ```
-       2. Create the service account and role by running:
-
-          ```
-          kubectl create -f rbac-config.yaml
-          ```
+    ```
+    kubectl config set-cluster <var>DESIRED_CLUSTER</var>
+    ```
+
+    To see the available clusters, run `kubectl config get-clusters`
+
+    2. Make sure that `kube-state-metrics` is installed on your machine:
+
+    ```
+    kubectl get deployment --all-namespaces | grep kube-state-metrics
+    ```
+
+    If it's not installed, follow the [instructions in the kube-state-metrics GitHub
+    repo](https://github.com/kubernetes/kube-state-metrics#kubernetes-deployment) to install it.
+
+    3. If your cluster uses role-based access, create a service account for `tiller` (Helm's service which runs inside
+    the Kubernetes cluster) using a Kubernetes manifest file:
+
+    4. Paste the following in a manifest file `rbac-config.yaml`:
+
+    ```
+    apiVersion: v1
+    kind: ServiceAccount
+    metadata:
+    name: tiller
+    namespace: kube-system
+    ---
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRoleBinding
+    metadata:
+    name: tiller
+    roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: ClusterRole
+    name: cluster-admin
+    subjects:
+    kind: ServiceAccount
+    name: tiller
+    namespace: kube-system
+    ```
+    2. Create the service account and role by running:
+
+    ```
+    kubectl create -f rbac-config.yaml
+    ```
     3. Initialize Helm so that `tiller` is installed in the cluster:
 
-       ```
-       helm init --history-max 200
```

Comment:
I believe that setting up helm should not be part of this guide

Reply:
Agreed, specially for helm 2. I'm stripping it.